client/state/selectors/get-user-setting.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Given a settingName, returns that setting if it exists or null
+ *
+ * @param  {Object} state Global state tree
+ * @param  {String} settingName - setting name
+ * @return {*} setting name value
+ */
+export default function getUserSetting( state, settingName ) {
+	const { settings, unsavedSettings } = state.userSettings;
+	let setting = null;
+
+	// If we haven't fetched settings, or if the setting doesn't exist return null
+	if ( settings && settings[ settingName ] !== undefined ) {
+		setting = unsavedSettings[ settingName ] !== undefined
+			? unsavedSettings[ settingName ]
+			: settings[ settingName ];
+	}
+
+	return setting;
```

Comment:
The desired behavior is to return `null` also in case when there is an unsaved value, but the original value is not set. I'll use `_.get` to clean up the conditional a bit.